### ISSUE TITLE
Custom Row Layouts won't save in versions > 2.28.0

### ISSUE THREAD

**Issue by ingageco** [siteorigin/siteorigin-panels#1182]
In versions above 2.28.0, when changing a Row Layout to a custom one added via filter, the layout does not save. It reverts to Standard.

**Comment by ingageco** [comment]
@AlexGStapleton  - absolutely. I'll get it posted here ASAP. Thank you

**Comment by ingageco** [comment]
@AlexGStapleton - So sorry - i'm not getting my GH notifications for some reason. I'll check but i do expect you are likely correct. However, since 2.25 < 2.28, ill test myself and report back as soon as I'm able.

**Comment by ingageco** [comment]
@AlexGStapleton sorry to dig this up, but i did narrow it down to this commit: 

https://github.com/siteorigin/siteorigin-panels/commit/5747123423a3dbce5a0d2cfaae24e5c921751119

TLDR: it's the migration of the old full-stretched and full-stretched-padded row_stretch options. I'm still using those, and they were removed after 2.28.0. 

Is there a hook that you know of for retrieving the panel data pre-editing so i can change the names of those?  I can manage either way, but that would make things easier.
